core/server/worker/src/test/java/alluxio/worker/netty/WriteHandlerTest.java

```diff
@@ -113,6 +113,36 @@ public void writeInvalidOffsetLaterRequest() throws Exception {
   }
 
   @Test
+  public void writeTwoRequests() throws Exception {
+    // Send first request
+    mChannel.writeInbound(newWriteRequest(0, newDataBuffer(PACKET_SIZE)));
+    mChannel.writeInbound(newEofRequest(PACKET_SIZE));
+    // Wait the first packet to finish
+    Object writeResponse = waitForResponse(mChannel);
+    checkWriteResponse(PStatus.OK, writeResponse);
+    // Send second request
+    mChannel.writeInbound(newWriteRequest(0, newDataBuffer(PACKET_SIZE + 1)));
+    mChannel.writeInbound(newEofRequest(PACKET_SIZE + 1));
+    writeResponse = waitForResponse(mChannel);
+    checkWriteResponse(PStatus.OK, writeResponse);
+  }
+
+  @Test
+  public void writeCancelAndRequests() throws Exception {
```

Comment:
Shall we have another test for `abort`?

Reply:
abort is not a server side control message. it is initiated from server side.
From the server side, only two types of messages: write or cancel